pkgs/firehose/lib/src/github.dart

```diff
@@ -223,3 +229,23 @@ class RpcException implements Exception {
   @override
   String toString() => 'RpcException: $message';
 }
+
+class GitFile {
+  final String filename;
+  final FileStatus status;
+
+  GitFile(this.filename, this.status);
+
+  String get relativePath =>
+      path.relative(filename, from: Directory.current.path);
+}
+
+enum FileStatus {
```

Comment:
It might be smart to incorporate this knowledge in the coverage checks...

Reply:
From a quick google, the `gh` is available from workflows. So you could get the list of changed files via `gh pr view 2615 --json files --jq '.files.[].path'`. There's probably other similar incantations that will give you more granularity (added, changed, removed, ...).

Reply:
Nevermind - I see we're already getting this information via the `listFilesForPR()()` call.